src/components/main/main.scss

```
@@ -1,3 +1,297 @@
+    background-color: #EEEEEE;
+    border-radius: 8px;
+    font-size: 15px;
+    padding: 0 35px;
+}
+.inputs-boxs::placeholder{
+    font-size: 15px;
+    color: #6b6b6b;
+}
+.first-input, .second-input{
+    position: relative;
+    width: 348px;
+}
+.svg-input-location{
+    position: absolute;
+    top: 10px; right: 15px;
+    border: none;
+    background: transparent;
+    outline: none;
+}
+.svg-input{
+    position: absolute;
+    top: 10px; right: 320px;
+    border: none;
+    background: transparent;
+    outline: none;
+}
+.vertical-line{
+    position: absolute;
+    width: 1px; /* Line width */
+    background-color: $primarycolor; /* Line color */
+    height: 40px; /* Override in-line if you want specific height. */
+    float: left; 
+    z-index: 2;
+    margin-top: 28px;
+    margin-left: 17px;
+}
+.style-click{
+    background-color: #fff;
+    border: solid 1px $primarycolor;
+}
+.buttons-download{
+    display: flex;
+    width: 100%;
+    height: 45vh;
+    overflow-x: hidden;
+}
+.container-buttons-download{
+    background-color: #EEEEEE;
+    width: 100%;
+}
+.title-content-buttons{
+    color: $primarycolor;
+    font-size: 35px;
+    font-family: 'Helvetica Neue', sans-serif;
+}
+.boxs-button-download{
+    display: flex;
+    gap: 30px;
+    width: 100%;
+    height: 100%;
+}
+.content-buttons-download{
+    padding: 30px 230px 30px 230px;
+}
+.first-button-download, .second-button-download{
+    position: relative;
+    display: flex;
+    background-color: #fff;
+    width: 560px;
+    margin-top: 30px;
+    cursor: pointer;
+}
+.img-download-desktop{
+    width: 150px;
+    height: 150px;
+}
+.img-download-mobile{
+    width: 84px;
+    height: 84px;
+}
+.texts-buttons-download{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin-left: 30px;
+    width: 100%;
+
+}
+.title-button-download{
+    color: $primarycolor;
+    font-family: 'Helvetica Neue', sans-serif;
+    font-size: 25px;
+    width: 100%;
+}
+.subtitle-button-download{
+    display: flex;
+    width: 100%;
+    justify-content: flex-start;
+    color: $primarycolor;
+    font-size: 12px;
+}
+.icon-arrow{
+    position: relative;
+    display: flex;
+    justify-content: flex-end;
+    align-content: center;
+    flex-wrap: wrap;
+    width: 100%;
+    margin-right: 25px;
+    right: 0;
+    bottom: 30px;
+}
+.img-download-mobile{
+    display: none;
+}
+@media screen and (max-width : 1024px){
+    .image-content-initial-desktop, .image-second-content-desktop, .image-third-content-desktop, .image-fourth-content-desktop{
+        display: none;
+    }
+    .image-content-initial-mobile, .image-second-content-mobile, .image-third-content-mobile, .image-fourth-content-mobile{
+        display: block;
+    }
+    .content-box, .second-content-box, .third-content-box, .fourth-content-box{
+        flex-direction: column;
+        padding: unset;
+        padding: 0 24px 0 24px;
```

```suggestion
        padding: 0 24px;
```

Isso funciona pra topo e lado, da mesma forma que foi feito acima